entity/device.go

```diff
@@ -3,22 +3,23 @@ package entity
 import "net"
 
 type Device struct {
```

Comment:
This gist can help you fix this issue: https://gist.github.com/up1/69a6749df9464f15743dbb1a0c711762

Reply:
@skatsaounis It worked thanks.